vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -75,6 +76,22 @@ var (
 	bookPathTemplate = gax.MustCompilePathTemplate("shelves/{shelf}/books/{book}")
 )
 
+func idempotentRetryCodes() []gax.CallOption {
+	return []gax.CallOption{
+		gax.WithRetryCodes([]codes.Code{
+			codes.DeadlineExceeded,
```

Comment:
I meant that I hope the retry logic doesn't automatically time.Sleep whenever it sees one of these codes. That would be a waste if the upstream call timed out.

Anyway, not really relevant for this PR.